docs/docs/styling.md

```diff
@@ -146,6 +146,29 @@ Navigation.mergeOptions(this.props.componentId, {
       enabled: true
     }
   },
+  fab: {
+    id: 'fab',  // required
+    backgroundColor: 'green',
+    clickColor: 'blue',
+    rippleColor: 'yellow',
+    visible: true,
+    icon: require('add.png'),
```

Comment:
This has no effect when you have an actions, or i missed something to make this work?

Reply:
![grafik](https://user-images.githubusercontent.com/24271650/64175787-5b903000-ce5c-11e9-87b2-3f7bfd2723aa.png)
Clickcolor is working for me, but the rest not.
I will take a look at it later.